Test/Update.test.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import addTask from "../modules/addTask.js";
+import completedTask from "../modules/completedTasks.js";
+import updateTask from "../modules/updateTask.js";
```

Comment:
Can we change this single quote? 🤔 This is raising some eslint errors . See [the actions tab](https://github.com/LYANGEND/to-do-list-preview/pull/1/checks) for more details.

Pls let do this for all occurrences. 🙏